### DISCUSSION TITLE
2023 Akash Events Resourcing Proposal

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#99] [Governance Proposals]
### Summary
Akash Network participation at large (eg - 1000+ attendee) In-person events are an important marketing, engineering, business development, community, investor relations and relationship building tactic.  However, event attendance is not a perfect science and we do not have an unlimited budget for events. With this being the case we started an Events Working Group in late January 2023 with members of the Overclock labs team, Akash Insider's and greater community representatives to debate, discuss and define what events are of significant importance for Akash to have a presence at in 2023. 

### TLDR : Proposal Development
With a working group formed we now had a team of community members to help define Akash representation at 2023 Events. Below are the high-level steps we used to get to this point.

1. [2023 Events Working Group](https://discord.gg/tsYj56j4UV) formed
2. 2023 Events Proposal drafted
3. Community Events Proposal meeting [video](https://drive.google.com/file/d/1uvPI2-yYk2YmDK9YkfiNw9hFkJCSM5_M/view?usp=sharing)
4. Proposal updated to include community feedback
5. [Current Proposal](https://docs.google.com/spreadsheets/d/1hfPnz3dS61_pomux93RhsyvHshaGHywHl07Cyi_M88s/edit#gid=580542914) Live for review

### Deep Dive : Proposal Development

**Event Selection**
As a working group, we selected which events to attend through the filter of three categories.
- Cloud events
- Crypto events (established crypto events e.g. - been around for more then a few years.)
- Cosmos events

Once these categories were established we reviewed worksheets listing all events of importance in 2023 [Example](https://docs.google.com/spreadsheets/d/1KtyFTb_W282bQ1xoVA5rlTwTDz3QfhswWIVpXhpbQIc/edit#gid=696686482) and filtered down to community defined important events for Akash.

At this point we refined even more to limit any event where the above three categories we're not included. 

This left us with 11 total events to attend in 2023.

**Budget**
Once we knew which events the working group wanted to attend, we needed to define the budget needed from the community pool to attend these events. We defined event participation and cost into two categories.

_Tier One events_ - These events will include a sponsorship to the main event or a side event and seperate Akash owned meetup.
Estimated Cost: $42,651/event

_Tier Two events_ - These events will not include a sponsorship or an Akash owned meetup.
Estimated Cost: $23,751/event

From there we were able to estimate the total cost of all events the community has defined Akash should attend in 2023. 

_Highly Suggested events_ - Events where if there were no Akash presence it could be potentially harmful to the brand and the community.
Estimated Cost: **$265,608**

_Total event costs_ - Events that would not be harmful if missed, but would still be of massive benefit to attend. 
Estimated Cost: **$360,612**

_We also included event overage_ - e.g. events that we don't know about yet, but would potentially need budget for attending. 
Estimated Cost: **$47,502**

**How things will get paid**
If approved, the total budget will be put into a community wallet that Overclock labs will manage. The wallet information will be public and reported out quarterly. 

Members of the Overclock labs team will expense cost of event participation and be paid out through a standardized system currently used.

Community members that attend events will be given an estimated amount of AKT equal to USD for each event they attend. 
Example:
Airfare - $400
Hotel - $1000
Food - $500
Total amount given to community attendee after event completion: $1,900 USD paid in AKT. 

You can see individual event costs in the [full proposal document here](https://docs.google.com/spreadsheets/d/1hfPnz3dS61_pomux93RhsyvHshaGHywHl07Cyi_M88s/edit#gid=580542914)

The final consideration is who from the community will attend these proposed events. This will be decided case by case for each approved event listed in the proposal sheet. These community member selection discussions will happen openly in the WG-Events working group channel in Discord.

As currently defined we would include 4 community representatives for each event Akash participates in.
- Event Manager
- Speaker
- Biz/Dev 
- Community support

Full definitions of each role can be found in the [event proposal document](https://docs.google.com/spreadsheets/d/1hfPnz3dS61_pomux93RhsyvHshaGHywHl07Cyi_M88s/edit#gid=268068160)

### Next Steps
With the 2023 events season underway, we are in need of a community governance vote to approve community pool resourcing for Akash event attendance. 

My proposal is to include the **highly suggested events + event overage** in the resources allocated for Akash events. The total resources allocated from the community pool will then be **$313,110** for the year 2023. 

**Personal note:** While the proposed allocation would be $313,110 from the community pool, from my experience running events for over ten years, I project the total cost for Akash events to be much less then that. As stated earlier in this proposal, events are not a perfect science and each event will require different needs. But having a budget to work from will allow us to actively participate in community activities we will otherwise miss out on.

Looking forward to feedback!

**Comment by Aihonaken** [comment]
This is a great proposal ! We need more exposure on Akash in the cosmos and worldwide. I have a few questions to the proposal. 
1. Are you panning to take a 1 time payment for the proposal, or a separate payment for each event ? 
2. Who is going to be in charge of the budget, and the expenses. 
3. The wallet that proposal budget is coming to is it a multisig wallet, or just a wallet that only one member of the WG is holding? 

With the AKT Volatility, wouldn't it be better to get the money from the pool separately for the different event. Since now $313,110k is more than 10% of the Community pool. But the situation can change drastically over the year. 

Overall, i like the proposal, great job on it!

**Reply by CaptainSplince** [reply]
Thanks @Aihonaken - Let me answer your questions with my thoughts.

Are you panning to take a 1 time payment for the proposal, or a separate payment for each event ?
>> Correct. The total will go into a budget we will use for up coming event participation and withdrawn for each event's needs.

Who is going to be in charge of the budget, and the expenses.
>> I will manage the budget and make the expenses public after events are over.

The wallet that proposal budget is coming to is it a multisig wallet, or just a wallet that only one member of the WG is holding?
>> I suggest taking the funds out and placing in a seperate multisig wallet. Funds will be taken out case by case from the "events wallet" for each event and the usage of those funds will all be made public. All funds that are not used within the year will go back into the community treasury.

**Comment by CaptainSplince** [comment]
I've updated the proposal above with information on
- Budget allocation
- Event attendee selection
- Community event attendee payment
- Budget transparency

**Reply by Aihonaken** [reply]
This is a great proposal so far! Voting Yes on the proposal ) 
thnx for clarifying all the concerns on the community sig.

**Comment by baktun14** [comment]
Hey Adam, great proposal. 

I was thinking, can some of these costs can be absorbed by the "Vendors & Marketing" genesis tokens?

Also, can the proposal be split into multiple proposals to help reduce the stress on the community pool like we did?

Thanks

**Reply by baktun14** [reply]
How do you measure the results of a sponsorship? What is usually included in a sponsorship that costs 20k for example?

**Reply by CaptainSplince** [reply]
Good Questions. 

In the proposal doc, there is a "forecasted outcomes" tab that lists out projects outcomes.

As for Vendor & Marketing tokens, those are for specific marketing vendors eg - CRM tooling, PR agency or advertising used for marking purposes. Events don't really require a tool other then attendance and participation.

**Comment by hewi333** [comment]
# Q4 Events Update 

### There will be no Governance Proposal for Q4 because the amount of funding remaining is sufficient to cover the projected events through the end of the year.

**Please see actual costs for the 2023 Events Budget [here](https://lookerstudio.google.com/u/0/reporting/c3273ae3-3749-46fe-8aae-4db6169cedf6/page/BIUND).**

Total Projected for 2023: $338,566
Actual Spend: $171,500.18*
Total AKT off-ramped: $143,710.46

**Actuals are still coming in from Messari and Cosmoverse.*

**Ideally, Actual Spend = Total off-ramped**

Unspent AKT: 48,963.77 or $42,451.59 at current prices
Off-ramped + Unspent AKT: $186,162.05
Projected to be off-ramped (Actual Spend - Total off-ramped): $27,789.72
Projected amount returned to the Community Pool: $14,661.87

These are projections based on a snapshot in time. With the volatility of AKT, the numbers may change. By the end of the year, members of Overclock Labs anticipate providing a more in-depth report on the first year of using Community Pool funds for events. We look forward to hearing any feedback from the community.

**Comment by hewi333** [comment]
Leftover 2023 Events funds sent back to the Community Pool [here](https://www.mintscan.io/akash/tx/13A493BA1998A871D80C87387A852896A07DCD770D22FF3ABB15AD5C9C9BFA6F?height=13870718).

**Comment by SophoclesSummer** [comment]
This proposal was approved on 4/4/2023 by a 83% majority. 
https://www.mintscan.io/akash/proposals/187
